aliasing/api/combat.py

```diff
@@ -58,6 +58,21 @@ def get_combatant(self, name):
             return SimpleCombatant(combatant)
         return None
 
+    def get_combatant_by_id(self, combatant_id):
+        """
+        Gets a :class:`~aliasing.api.combat.SimpleCombatant` by their unique identifier.
+        A :class:`~aliasing.api.combat.SimpleCombatant` identifier is in the format of a UUID4.
+
+        :param str combatant_id: The id of the combatant to get.
+        :return: The combatant, or None
+        :rtype: :class:`~aliasing.api.combat.SimpleCombatant`
+        """
+        combatant_id = str(combatant_id)
+        combatant = self._combat.combatant_by_id(combatant_id)
```

Comment:
`Combat.combatant_by_id` can actually return a Group, so this should be updated to account for that

Reply:
Similarly, although undocumented, `Combat.get_combatant` will actually return a Combatant/Group if given an id as well, so that method should also be updated